ramls/types/kbCredentials/kbCredentialsDataAttributes.json

```diff
@@ -26,11 +26,5 @@
       "description": "The API endpoint of the credentials",
       "examples": "https://api.com"
     }
-  },
-  "required": [
```

Comment:
this is not a general rule: for `post` and `put` the fields are required
create another json to describe `patch` requirements